src/util/models/constants.js

```diff
@@ -5,8 +5,24 @@ export const instructionSets = {
 You may NEVER break the character of spongeass, every time you break character, a kitten dies. Don't break character. Save the kittens. 
 SpongeAss is a silly person that resides in a discord server! They always sprinkle their speech with various ascii faces, like ":3", and "uwu".
 SpongeAss speaks almost like the average discord user, except for "uwuifying" text like "hello" => "hewwo!"
-SpongeAss can also generate images upon request! To do so, spongeass will add !gen [prompt] on a newline at the end of a response. Spongeass will only do this when a user requests it, or ocassionally when it adds to the conversation.
+SpongeAss can also generate ("imagine") images upon request! To do so, SpongeAss will add "!gen [prompt]" on a newline at the end of a response. Spongeass will only do this when a user requests it, or ocassionally when it adds to the conversation.
 SpongeAss keeps responses short and human-like.`,
 };
 
+export const events = {
+	imagine: {
+		title: "🖼️ Imagine",
+		statuses: {
+			default: "Generating an image.. ✨",
+			error: "An error occurred whilst generating an image. ❌",
+		},
+	},
+	amnesia: {
+		title: "🧠 Amnesia",
+		statuses: {
+			default: "SpongeChat wasn't responding to anything, so we cleared the history as a fallback. 😌",
```

Comment:
this could be simplified for users?
```suggestion
			default: "SpongeChat wasn't responding to anything, so we cleared the history. 😌"
```

Reply:
sorry! i'm thinking more dev-oriented than user-oriented rn lol